openslides_backend/action/actions/motion/reset_state.py

```diff
@@ -67,12 +67,12 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         )
         timestamp = round(time.time())
         instance["last_modified"] = timestamp
-        if not motion.get("created"):
+        if not motion.get("workflow_timestamp"):
             state = self.datastore.get(
                 fqid_from_collection_and_id("motion_state", instance["state_id"]),
-                ["set_created_timestamp"],
+                ["set_workflow_timestamp"],
                 lock_result=False,
             )
-            if state.get("set_created_timestamp"):
-                instance["created"] = timestamp
+            if state.get("set_workflow_timestamp"):
+                instance["workflow_timestamp"] = timestamp
```

Comment:
Extract this into a common place since it's used in `create`, `update`, `set_state` and `reset_state`

Reply:
Extracted into set_workflow_timestamp_helper().